src/components/modules/VehicleList/index.jsx

```diff
@@ -1,65 +1,27 @@
 import React from 'react';
+import PT from 'prop-types';
+import { withRouter } from 'react-router-dom';
+
+import vehicleData from 'services/data/vehicles';
 
 import PageHeaderBarContainer from 'common/PageHeaderBarContainer';
 
 import VehicleListItem from './VehicleListItem';
 
-const mockData = {
-  huubSpot: {
-    name: 'Schoonschip',
-    city: 'Amsterdam',
-    vehicleTypes: [
-      {
-        vehicleType: 'middenklasser',
-        vehicles: [
-          {
-            vehicleId: 'ABC-1-A',
-            mainBatteryPercentage: 91,
-            lockBatteryPercentage: 53,
-          },
-          {
-            vehicleId: 'AAC-2-B',
-            mainBatteryPercentage: 64,
-            lockBatteryPercentage: 97,
-          },
-          {
-            vehicleId: 'ABB-1-C',
-            mainBatteryPercentage: 65,
-            lockBatteryPercentage: 73,
-          },
-          {
-            vehicleId: 'BAB-3-C',
-            mainBatteryPercentage: 20,
-            lockBatteryPercentage: 46,
-          },
-          {
-            vehicleId: 'CCA-2-A',
-            mainBatteryPercentage: 87,
-            lockBatteryPercentage: 20,
-          },
-        ],
-      },
-    ],
-  },
-};
-
 class VehicleList extends React.Component {
 
   render() {
-    const {
-      huubSpot: {
-        name,
-        vehicleTypes,
-      },
-    } = mockData;
+    const { match: { params } } = this.props;
+    const huubSpotName = vehicleData[params.huubSpot].huubSpotName;
+    const vehicleList = vehicleData[params.huubSpot].vehicles[params.vehicleType];
```

Comment:
This is dangerous, as you rely on the url. I would at least check whether these props of `params` exist before using them.